cms/templates/activation_complete.html

```diff
@@ -9,8 +9,8 @@
 <div class="wrapper-mast wrapper sr">
   <header class="mast">
     <h1 class="page-header">
-      ${_("{studio_name} Account Activation").format(
-        studio_name=Text(settings.STUDIO_SHORT_NAME)
+      ${Text(_("{studio_name} Account Activation")).format(
+        studio_name=Text(settings.STUDIO_SHORT_NAME),
```

Comment:
@douglashall yes, you need `Text` here. I made the mistake of not including it earlier.

When wrapping a string in `Text` or `HTML`, you're generating a markupsafe string which has an overloaded `format` method. The interpolation of `Text` and `HTML`-wrapped strings into the top level string only works when that top-level string itself is a markupsafe string.